requirements.txt

```diff
@@ -1 +1,3 @@
-requests >= 2.31.0
\ No newline at end of file
+requests >= 2.31.0
+fastapi==0.109.0
+uvicorn==0.26.0
```

Comment:
It's probably better to have a `>=` requirement here and perhaps an upper bound to pull the latest security/bug fixes into every build

Reply:
by setting an upper bound, how will i know how high to set it?